thinc/layers/dropout.py

```diff
@@ -9,16 +9,51 @@
 
 
 @registry.layers("Dropout.v1")
-def Dropout(rate: float = 0.0) -> Model[InT, InT]:
+def Dropout(
+    rate: float = 0.0
+) -> Model[InT, InT]:
     """Help prevent overfitting by adding a random distortion to the input data
     during training.  Specifically, cells of the input are zeroed with
     probability determined by the `rate` argument.
     """
-    return Model("dropout", forward, attrs={"dropout_rate": rate, "is_enabled": True})
+    return Model(
+        "dropout",
+        forward,
+        attrs={
+            "rate_attr_name": "dropout_rate",
```

Comment:
Alternatively, we could call this `dropout_attr`, which would also work as a name for other models if we decide to have this feature across the code base.

Reply:
I think it would make the code more readible than `rate_attr_name`.